pkgs/by-name/qu/quarkus/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchurl
+, writeScript
+, bash
+, openjdk
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "quarkus-cli";
+  version = "3.8.3";
+
+  src = fetchurl {
+    url = "https://github.com/quarkusio/quarkus/releases/download/${finalAttrs.version}/quarkus-cli-${finalAttrs.version}.tar.gz";
+    hash = "sha256-+HUVnd21q2xln58UR1QS0UFCjlY0sbf79ZkvWbRCH7Q=";
+  };
+
+  launcher = writeScript "quarkus" ''
+    #! ${bash}/bin/bash
```

Comment:
```suggestion
  launcher = writers.writeBash "quarkus" ''
```

Reply:
Actually, bash is not really needed. Can writeScriptBin be enough ?

Reply:
I saw that makewrapper could do the trick without having to export vars. But AS quarkus cli requires openjdk, is it a good idea to use it ?

Reply:
> Actually, bash is not really needed. Can writeScriptBin be enough ?

Looks like it doesn't work with `writeScriptBin`

Reply:
Thatś what I saw. I actually managed to correctly use `makewrapper`. I am building the package right now.
It will be a lot less of hacky things